src/store/contacts/contacts-state.ts

```diff
@@ -0,0 +1,16 @@
+import { AccountBaseInterface } from '../../interfaces/account.interface';
+
+export interface ContactsState {
+  contacts: Array<AccountBaseInterface>;
+  ignoredAddresses: Array<string>;
+  contactCandidatePkh: string;
+}
+export const contactsInitialState: ContactsState = {
+  contacts: [],
+  ignoredAddresses: [],
+  contactCandidatePkh: ''
+};
+
+export interface ContactsBookRootState {
```

Comment:
check all file names inside `src/store/contacts/` - should be "contact book` right now